Mathlib/RingTheory/LaurentSeries.lean

```diff
@@ -33,10 +35,41 @@ abbrev LaurentSeries (R : Type*) [Zero R] :=
   HahnSeries ℤ R
 #align laurent_series LaurentSeries
 
-variable {R : Type u}
+variable {R : Type*}
 
 namespace LaurentSeries
 
+section Zero
+
+variable [Zero R]
+
+-- generalize to any locally finite linear order? see #18604
```

Comment:
I suppose this is referring to a mathlib3 PR?

Reply:
Yes, that's right.